db/migrate/helpers/delete_content_item.rb

```diff
@@ -1,40 +1,8 @@
 module Helpers
   module DeleteContentItem
     def self.destroy_content_items_with_links(content_ids)
-      content_ids = Array(content_ids)
-
-      content_items = ContentItem.where(content_id: content_ids)
-      destroy_supporting_objects(content_items)
-      content_items.destroy_all
-
-      destroy_links(content_ids)
-    end
-
-    def self.destroy_supporting_objects(content_items)
```

Comment:
If we remove this method we should remove it from any migrations and update documentation